x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/actions/get_action.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { IClickActionDescriptor } from '../';
+import { ACTION, ActionComplete, UISession } from '../../../../../common/search/sessions_mgmt';
+import extendSessionIcon from '../../icons/extend_session.svg';
```

Comment:
@restrry This PR has starting showing a lot of TS failures after merging master today. This line gives one example:
```
x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/actions/get_action.tsx:11:31 - error TS2307: Cannot find module '../../icons/extend_session.svg' or its corresponding type declarations.

11 import extendSessionIcon from '../../icons/extend_session.svg';
```
The file path is valid, but it seems TS doesn't know about it. Is there something I should change in the tsconfig.json file for data_enhanced to make this custom icon work?

Other than this area, there are dozens of errors relating to referencing files that are outside of `data_enhanced`- for example, Management. I think this PR is blocked until the referenced projects are converted to use their own tsconfig.json file. Is that correct?